### PR TITLE
add permaswap in banker

### PR DESCRIPTION
**Pull request by xiaojay**

### CODE REVIEW

File: process/npc/LlamaBanker.lua [hunk 1]
```diff
@@ -374,7 +374,7 @@ Handlers.add(
     print('Account: ' .. account .. ', Balance: ' .. balance)
 
     local message = 'Psst, you currently have ' .. FormatLlamaTokenAmount(balance) .. ' $LLAMA coins!'
-    message = message .. ' Go to ao-bazar.g8way.io to view and trade your ao tokens.'
+    message = message .. ' Go to ao-bazar.g8way.io or www.permaswap.network/#/ao/LLAMA-AR to view and trade your ao tokens.'
```
Comment (elliotsayes) [inline process/npc/LlamaBanker.lua]: How about just “permaswap.network”? Main page should be enough I think

**Comment by elliotsayes** [comment]
Good call :) I’ll add this in when I’m at my laptop

File: process/npc/LlamaBanker.lua (hunk 1 above)
Comment (xiaojay) [reply process/npc/LlamaBanker.lua]: ok, updated

**Comment by elliotsayes** [comment]
Now we got two :)
Screenshot 2024-07-29 at 5 49 27 AM
Screenshot 2024-07-29 at 6 07 36 AM

**Pull request merged**